src/main/java/codelook/jpa/objects/AuthorInfoRepo.java

```diff
@@ -0,0 +1,13 @@
+package codelook.jpa.objects;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface AuthorInfoRepo extends CrudRepository<AuthorInfo, Integer> {
```

Comment:
Consider changing to JPARepository #23

Reply:
Change to jpa repos for all repositories has been committed and pushed